src/c2goto/CMakeLists.txt

```diff
@@ -38,19 +38,21 @@ endif()
 
 function(mangle_clib output)
   set(result "")
-   file(GLOB c2goto_library_files CONFIGURE_DEPENDS
+  # Extract a list of C's operational models
+  file(GLOB c2goto_library_files CONFIGURE_DEPENDS
     "${CMAKE_CURRENT_SOURCE_DIR}/library/*.c")
+  # Extract a list of C's math operational models
   file(GLOB c2goto_libm_files CONFIGURE_DEPENDS
     "${CMAKE_CURRENT_SOURCE_DIR}/library/libm/*.c")
   file(GLOB_RECURSE c2goto_header_files CONFIGURE_DEPENDS
     "${CMAKE_CURRENT_SOURCE_DIR}/headers/*.h")
+
+  # For each config, generate clib*.c and clib*.goto file in the current binary directory
   foreach(in_f ${ARGN})
     set (in_f "${${in_f}}")
-    list(GET in_f 0 in_file)
+    list(GET in_f 0 out_file)
```

Comment:
Isn't this going to break relative paths? We should at least add an assertion that only absolute paths are being used in the list.

Reply:
>Isn't this going to break relative paths?

No, it won't break the path. 

The goal here is to create `clib32/64.c` in `build/src/c2goto/`. The `out_file` variable is pointing to `clib32/64.c` and the current binary directory is already `build/src/c2goto/`. I don't think we need the relative path or assertion for absolute path at all in the first place.